src/libs/Notification/PushNotification/subscribeToPushNotifications.ts

```diff
@@ -37,23 +63,9 @@ Onyx.connect({
     },
 });
 
-function getLastUpdateIDAppliedToClient(): Promise<number> {
-    return new Promise((resolve) => {
-        Onyx.connect({
-            key: ONYXKEYS.ONYX_UPDATES_LAST_UPDATE_ID_APPLIED_TO_CLIENT,
-            callback: (value) => resolve(value ?? CONST.DEFAULT_NUMBER_ID),
-        });
-    });
-}
-
 function applyOnyxData({reportID, reportActionID, onyxData, lastUpdateID, previousUpdateID, hasPendingOnyxUpdates = false}: ReportActionPushNotificationData): Promise<void> {
     Log.info(`[PushNotification] Applying onyx data in the ${Visibility.isVisible() ? 'foreground' : 'background'}`, false, {reportID, reportActionID});
 
-    if (!ActiveClientManager.isClientTheLeader()) {
```

Comment:
This code only runs on native and this check is always `true` on native so this is actually dead code.